src/app/api/core/utils/authenticate.ts

```diff
@@ -16,7 +16,12 @@ export const _authenticateWithToken = async (token: string): Promise<User> => {
   }
 
   // Access to IU only. The reason for not checking the IUID directly is for the webhook events to pass
-  if (payload.data.clientId || payload.data.companyId) {
+  if (
+    payload.data.clientId ||
+    (payload.data.companyId &&
+      payload.data.internalUserId &&
+      payload.data.companyId != 'default') // allow case when the parsed token have companyId = "default"
+  ) {
```

Comment:
```suggestion
  if (
   !payload.data.internalUserId
  ) {
```

I think this works, doesn't it?